src/app/components/source/UserInfo.js

```diff
@@ -73,24 +74,27 @@ const UserInfo = (props) => {
             defaultMessage="{teamsCount, plural, one {# workspace} other {# workspaces}}"
             description="Count of how many work spaces this user account can access"
             values={{
-              teamsCount: props.user.team_users.edges.length || 0,
+              teamsCount: props.user.number_of_teams || 0,
```

Comment:
This should be a safe property access too, `props.user?.number_of_teams`

Reply:
Done